engine/src/main/resources/templates/drawer/Inventory/validationErrorBody.html

```diff
@@ -0,0 +1 @@
+If no hosts were created by this change, the error will not appear in the service.
```

Comment:
This template doesn't look very helpful.

Reply:
It's part of mocks provided by UX team, so I gess it could be helpful for the user ?

Reply:
There is useful information (host name and error message) in the email template this was extracted from.

This version is kind of empty and doesn't look like it'll help the user in any way.

Reply:
I checked with Jess, the idea is to show to tenants what drawer notifications could looks like.
Then they will have to adjust and validate them based on stage data.